app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
 
+    people = {}
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
+
+
+def create_person_list(people: list[dict]) -> list[Person]:
+    persons_ls = []
+    persons_ls = [Person(body["name"], body["age"]) for body in people]
+
+    for cha in range(len(people)):
```

Comment:
cha is not a descriptive variable name

no need to use indexing, you can iterate straight through  the elements  of the list